modules/ROOT/pages/update-acm.adoc

```diff
@@ -21,6 +21,11 @@ The staging environment will have a partial or full copy of the data in the prod
 
 To update your API community:
 
+. Ensure your Salesforce organizations have the "External Data Change Tracking for OData 4.0 Adapter" feature enabled.
+.. Before upgrading the package in your production or sandbox organization, check that your organization has this feature enabled by going to *Setup* > *External Data Source* and editing the Exchange data source.
+.. If you don't see a checkbox field with the label *Eligible for External Change Data Capture*, raise a support case through the MuleSoft support portal, and include the organization IDs of all your Salesforce API Community Manager organizations, including production and all sandboxes.
```

Comment:
```suggestion
.. If you don't see an *Eligible for External Change Data Capture* checkbox, raise a support case through the MuleSoft support portal and list the IDs of all your Salesforce API Community Manager organizations, including production and all sandboxes.
```
"including the IDs of all production and sandbox environments"?

Reply:
Fixed.